### ISSUE TITLE
The Example gives match score:0

### ISSUE THREAD

**Issue by hasansahinnn** [brianrho/fpmatch#3]
I took a few errors then fix all of them but I always take match score 0. I'm using your templates.
I want to take fingerprint template from device by using esp32 or ardunio, then send with api and match in there
Thanks for help

**Comment by brianrho** [comment]
As I said earlier, you need to send details about everything you've done. Which are you running, Python or C? How did you fix the errors? If you've modified the code, what does it look like now? What exactly was printed when you ran the code?

**Comment by hasansahinnn** [comment]
I used both. when i want to execute from c,i taked <sys/time.h>and WEBAPI errors. then I tried py code and I taked '%1 not Win32 application' error or something like that I don't remember right now. Anyway, I fixed both and successfully executed. The compare match is only 0. I found another ARTH_DLL in another project then opened with Visual code + install python 3.7 32 bit. It's working right now. I will share project when I finished.
